src/mac-now-playing/meson.build

```diff
@@ -0,0 +1,15 @@
+mac_now_playing_sources = [
+  'MacNowPlaying.mm'
+]
+
+
+shared_module('mac-now-playing',
+  mac_now_playing_sources,
+  dependencies: [audacious_dep],
+  name_prefix: '',
+  objc_args: ['-std=c99', '-fobjc-exceptions'],
+  objcpp_args: '-std=c++11',
```

Comment:
Remove the objc/objcpp args, they should not be required (our build pipeline should confirm this). And if they are, the Makefile should include them as well.